decidim-consultations/app/controllers/decidim/consultations/admin/consultations_controller.rb

```diff
@@ -64,6 +64,13 @@ def update
           end
         end
 
+        # GET /admin/consultations/:slug/results
+        def results
+          enforce_permission_to :read, :consultation, consultation: current_consultation
+          @total_votes = current_consultation.questions.reduce(0) { |sum, question| sum + question.total_votes }
```

Comment:
Not sure if ActiveRecord can handle it, but maybe?

```suggestion
          @total_votes = current_consultation.questions.sum(:total_votes)
```

Also, it might be better to move this to a method at `Consultation`?

Reply:
Thanks. My first locales contribution, I didn't know the rules yet. I'll move to a method that too.